src/main/resources/application.conf

```diff
@@ -139,6 +139,12 @@ grothendieck {
     # Doc: https://github.com/ethereum/EIPs/issues/150
     eip150-block-number = "2500000"
 
+    # EIP-155 fork block number
+    # Doc: https://github.com/ethereum/eips/issues/155
+    # 3 000 000 because of
+    #https://github.com/paritytech/parity/blob/b50fb71dd1d29dfde2a6c7e1830447cf30896c31/ethcore/res/ethereum/classic.json#L15
```

Comment:
Alternatively, we could have those description differ slightly here and in `blockchain.conf`